src/test/java/step3/domain/LottoTest.java

```diff
@@ -0,0 +1,38 @@
+package step3.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LottoTest {
+    private List<Integer> numbers;
+
+    @BeforeEach
+    void beforeEach() {
+        numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
+    }
+
+    @Test
+    void valid_로또_생성() {
+        Assertions.assertThat(new Lotto(numbers)).isEqualTo(new Lotto(numbers));
+    }
+
+    @Test
+    void invalid_로또_생성_숫자개수() {
+        assertThatThrownBy(() -> {
+            Lotto newLotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5));
+        }).isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("중복되지 않은 6개의 숫자를 입력해주세요.");
+    }
```

Comment:
사용하지 않는 변수는 제거해 줘도 좋을 것 같아요 😄 
```suggestion
    void invalid_로또_생성_숫자개수() {
        assertThatThrownBy(() -> new Lotto(Arrays.asList(1, 2, 3, 4, 5)))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessageContaining("중복되지 않은 6개의 숫자를 입력해주세요.");
    }
```

Reply:
수정하였습니다. 😄